app/assets/stylesheets/base/_reset.scss

```diff
@@ -31,5 +31,5 @@ a, button, input[type="submit"] {
 }
 
 .hidden {
-  display: none;
+  display: none !important;
 }
```

Comment:
Don't use important, is a bad practice and may create conflicts with other styles.

Reply:
yeah, that was the first thought i have but since the hidden it have less weight vs the display i have on other class then it overlaps ignoring the hidden, to solve that i chose between add more weight nesting the hidden class on the element or make that the class hidden take more weight with important, even with that, should i choose the other option?

Reply:
Well, it is not the best option and definitely you shouldn't make use of important.
You can reorder your CSS code or look for another way to do what you are doing, you could create another PR where you apply refactoring once you have your project finished.